.circleci/config.yml

```diff
@@ -169,6 +173,18 @@ jobs:
       - codecov/upload:
           file: cover_main.out
 
+  release:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          working_directory: ~
+          name: Download release app
+          command: go get github.com/itchio/gothub
```

Comment:
Why ci should know about lib you are using? If you will decide to change lib, you will need to change ci config and makefile.